ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -328,10 +336,33 @@ public BlobIdType getType() {
   /**
    * @return {@code true} if the blob that this id represents is encrypted. {@code false} otherwise
    */
-  public boolean isEncrypted() {
+  private boolean isEncrypted() {
     return isEncrypted;
   }
 
+  /**
+   * Check if encrypted bit in blobId is set based on original blobId string.
+   * @return {@code true} if encrypted bit in this string is set. {@code false} otherwise
+   */
+  public static boolean isEncrypted(String blobIdString) {
+    if (blobIdString == null) {
```

Comment:
Let us drop this check. If a caller calls with a null argument, there is an error somewhere, so perhaps better to let this result in an NPE.